cmd/veneur-emit/README.md

```diff
@@ -7,58 +7,66 @@ Some common use cases:
 
 # Usage
 
-`veneur-emit` can read an existing veneur [config file](https://github.com/stripe/veneur#configuration). If that's not convenient, you can specify it's configuration options directly.
+Emitting a metric with veneur-emit:
+
+```
+$ veneur-emit -hostport example.com:8125 -count 3 -name "my.test.metric" -tag "host:my.machine.local"
```

Comment:
should we make that address `udp://example.com:8125 ` instead, to demonstrate the thing? (:

Reply:
updated!